src/util.ts

```diff
@@ -49,10 +49,13 @@ export function errToString(error: any): string {
 	return error.toString();
 }
 
-export function showResult(result: string, column?: vscode.ViewColumn): Thenable<void> {
+export function showResult(result: string, filename: string, column?: vscode.ViewColumn): Thenable<void> {
 	let uri: vscode.Uri = null;
+	if (!filename.endsWith('.json')) {
```

Comment:
Remove this logic - we should always pass in a filename with '.json' if the file is supposed to end in 'json'